recipes/manta/meta.yaml

```diff
@@ -6,17 +6,28 @@ package:
   version: {{ version }}
 
 source:
-  url: https://github.com/Illumina/manta/releases/download/v{{ version }}/manta-{{ version }}.centos6_x86_64.tar.bz2
-  sha256: ae19b1b934cf5bb605dfb58b29e8e2b843cb469ec5ff12441ca3d9d39179abf4
+  url: https://github.com/Illumina/manta/releases/download/v{{ version }}/manta-{{ version }}.release_src.tar.bz2
+  sha256: c846d61b02483265c09d58bd85dacf5326a94f38179b5ae4f70694be96e1368f
+  patches:
+    - boost-shlibs.patch
 
 build:
-  number: 3
+  number: 4
   run_exports:
     - {{ pin_subpackage(name, max_pin="x") }}
+  skip: True  # [osx] # until CI os/x is fixed https://github.com/bioconda/bioconda-recipes/issues/52553
 
 requirements:
+  build:
+    - cmake
+    - make
+    - {{ compiler('cxx') }}
+    - boost-cpp
```

Comment:
this should be in `host`

Reply:
That fails to lint:
```
The recipe requests a compiler in a section other than build
Please move the `{{ compiler('language') }}` line into the`requirements: build:` section
```

Reply:
Sorry, I meant just `boost-cpp`.
`build` should list only build **tools**. All software _libraries_ should be in `host`

Reply:
Moving boost-cpp seems to be better - just trying that